kmod/src/item.c

```diff
@@ -2530,27 +2530,42 @@ void scoutfs_item_invalidate(struct super_block *sb, struct scoutfs_key *start,
 	put_pg(sb, right);
 }
 
+static unsigned long item_cache_count_objects(struct shrinker *shrink,
+					      struct shrink_control *sc)
+{
+	struct item_cache_info *cinf = container_of(shrink,
+						    struct item_cache_info,
+						    shrinker);
+	struct super_block *sb = cinf->sb;
+
+	scoutfs_inc_counter(sb, item_cache_scan_objects);
+
+	//FIXME just picked ->lru_pages here being dumb
+	return min_t(u64, (u64)(&cinf->lru_pages) * SCOUTFS_BLOCK_LG_PAGES_PER,
```

Comment:
The * LG_PAGES_PER only makes sense for the block cache.  Remove it from the others.

Reply:
Hmm, so can we get rid of the `min_t(u64, , ULONG_MAX / 2)` here then? Or do we still want to cap at ULONG_MAX / 2 for safety?

Reply:
I think we still want the cap, just so the counters don't wrap, though it does look like a pretty arbitrary and silly cap.